### PR TITLE
Wqp 746

### PR DESCRIPTION
**Pull request by mbucknell**

### CODE REVIEW

**Comment by mbucknell** [comment]
OpenLayers 2 is still in the project because it's still used for the coverage mapper page. I moved ol3 specific map utilities into the ol3Utils subdirectory.

File: portal_ui/static/js/MapConfig.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* jslint browser: true */
```
Comment (mbucknell) [inline portal_ui/static/js/MapConfig.js]: Moved defaults into a separate MapConfig.js that both the ol2 and ol3 code can access.

File: portal_ui/static/js/identifyDialog.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/* jslint browser: true */
```
Comment (mbucknell) [inline portal_ui/static/js/identifyDialog.js]: Now using Handlebars to create the html for the site info and hidden form inputs. This module handles both the UI dialog and the popover. The popover is used when the width of the screen is extra small.

File: portal_ui/static/js/identifyDialog.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+	var FEATURE_INFO_TEMPLATE = Handlebars.compile(
+		'{{#if exceedsFeatureLimit}}' +
+		'Retrieved {{features.length}} sites, only showing ' + FEATURE_LIMIT + '<br />' +
+		'{{/if}}' +
+		'{{#each features}}' +
+			'<br /><table>' +
+			'<tr><th>Station ID: </th><td class="details-site-id">{{values_.name}}</td></tr>' +
+			'<tr><th>Name: </th><td>{{values_.locName}}</td></tr>' +
+			'<tr><th>Type: </th><td>{{values_.searchType}}</td></tr>' +
```
Comment (dsteinich) [inline portal_ui/static/js/identifyDialog.js]: Is this the correct site type to show - it matches what was selected on the UI form. type is more descriptive.

Comment (mbucknell) [inline portal_ui/static/js/identifyDialog.js]: I'll double check with Jim.

**Pull request merged**